src/scss/atomic/organisms/_overview.scss

```diff
@@ -41,3 +41,28 @@
 .tooltip-enter-active, .tooltip-leave-active {
     transition: opacity $defaultDuration $defaultEasing, transform $defaultDuration, left 0s $defaultEasing;
 }
+
+
+.overview .cta__block {
```

Comment:
Can you make this a cta__block--<modifier> like cta__block--compact so it is reusable?